app/services/provider_interface/sort_application_choices.rb

```diff
@@ -9,7 +9,7 @@ def self.call(application_choices:)
     def self.for_task_view(application_choices)
       application_choices.from <<~WITH_TASK_VIEW_GROUP.squish
         (
-          SELECT a.*, c.recruitment_cycle_year,
```

Comment:
The `c.recruitment_cycle_year` we are dropping here is already referencing the recruitment cycle year of the current (not the original) course.